From 97578460809075ad82857ff876cc04738d93fdd7 Mon Sep 17 00:00:00 2001
From: Bart Ribbers <bribbers@disroot.org>
Date: Sun, 4 Apr 2021 12:52:12 +0200
Subject: [PATCH] Upgrade websocket-client to 0.58.0

core equivalent of https://github.com/MycroftAI/mycroft-messagebus-client/pull/21
There was an incompability with the latest websocket-client and the
messagebus which needed fixing for Linux distro compatibility. Since
messagebus-client was going to depend on websocket-client 0.58.0, let's
do the same in core
---
 mycroft/client/enclosure/base.py                            | 4 ++--
 requirements/requirements.txt                               | 4 ++--
 test/integrationtests/voight_kampff/features/environment.py | 4 ++--
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/mycroft/client/enclosure/base.py b/mycroft/client/enclosure/base.py
index 4d26a42f43fa..ae4ea7edcb63 100644
--- a/mycroft/client/enclosure/base.py
+++ b/mycroft/client/enclosure/base.py
@@ -493,7 +493,7 @@ def open(self):
         LOG.info('New Connection opened!')
         self.synchronize()
 
-    def on_close(self):
+    def on_close(self, _):
         LOG.info('Closing {}'.format(id(self)))
         GUIWebsocketHandler.clients.remove(self)
 
@@ -525,7 +525,7 @@ def synchronize(self):
                            })
             namespace_pos += 1
 
-    def on_message(self, message):
+    def on_message(self, _, message):
         LOG.info("Received: {}".format(message))
         msg = json.loads(message)
         if (msg.get('type') == "mycroft.events.triggered" and
diff --git a/requirements/requirements.txt b/requirements/requirements.txt
index c9ccedaf313b..5234739cf9c8 100644
--- a/requirements/requirements.txt
+++ b/requirements/requirements.txt
@@ -4,7 +4,7 @@ PyAudio==0.2.11
 pyee==8.1.0
 SpeechRecognition==3.8.1
 tornado==6.0.3
-websocket-client==0.54.0
+websocket-client==0.58.0
 requests-futures==0.9.5
 pyserial==3.0
 psutil==5.6.6
@@ -18,7 +18,7 @@ PyYAML==5.4
 lingua-franca==0.4.1
 msm==0.8.9
 msk==0.3.16
-mycroft-messagebus-client==0.9.1
+mycroft-messagebus-client==0.9.2
 adapt-parser==0.3.7
 padatious==0.4.8
 fann2==1.0.7
diff --git a/test/integrationtests/voight_kampff/features/environment.py b/test/integrationtests/voight_kampff/features/environment.py
index 662c91206919..12e092d9a18f 100644
--- a/test/integrationtests/voight_kampff/features/environment.py
+++ b/test/integrationtests/voight_kampff/features/environment.py
@@ -44,11 +44,11 @@ def __init__(self):
         self.message_lock = Lock()
         self.new_message_available = Event()
 
-    def on_message(self, message):
+    def on_message(self, _, message):
         with self.message_lock:
             self.messages.append(Message.deserialize(message))
         self.new_message_available.set()
-        super().on_message(message)
+        super().on_message(_, message)
 
     def get_messages(self, msg_type):
         with self.message_lock:
